src/components/CalendarDay.jsx

```diff
@@ -93,12 +89,9 @@ export default class CalendarDay extends React.Component {
 
     if (!day) return <td />;
 
-    const modifiersForDay = getModifiersForDay(modifiers, day);
-
     const className = cx('CalendarDay', {
       'CalendarDay--outside': isOutsideDay,
-    }, modifiersForDay.map(mod => `CalendarDay--${mod}`));
-
+    }, [...modifiers].map(mod => `CalendarDay--${mod}`));
```

Comment:
`Array.from(modifiers, mod => `CalendarDay--${mod}`)` is much much more efficient because it avoids the intervening array.